x-pack/plugins/triggers_actions_ui/public/application/app.tsx

```diff
@@ -92,26 +96,21 @@ export const renderApp = (deps: TriggersAndActionsUiServices) => {
 };
 
 export const App = ({ deps }: { deps: TriggersAndActionsUiServices }) => {
-  const { dataViews, theme } = deps;
+  const { dataViews, i18n, theme } = deps;
   const sections: Section[] = ['rules', 'logs'];
-  const isDarkMode = theme.getTheme().darkMode;
 
   const sectionsRegex = sections.join('|');
   setDataViewsService(dataViews);
   return (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it